Tests/ClientTests/CreditCard/CreditCardSingleViewModelTests.swift

```diff
@@ -0,0 +1,130 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+@testable import Client
+import Storage
+import Shared
+import XCTest
+
+class CreditCardSingleViewModelTests: XCTestCase {
```

Comment:
It's a great start the class has 49.7% of coverage which is shy to our minimum coverage per class I think there are func that can be tested and edge cases that are not covered yet